charts/renku/templates/tests/test-renku.yaml

```diff
@@ -1,51 +1,42 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: "{{ .Release.Name }}-test"
+  name: "{{ .Release.Name }}-acceptance-tests"
   annotations:
-    "helm.sh/hook": test-success
+    "helm.sh/hook": test
 spec:
-  initContainers:
-  - name: renku-demo
-    image: renku/renku-demo:0.3.0
-    env:
-      - name: GITLAB_URL
-        value: {{ template "http" . }}://{{ .Values.global.renku.domain }}{{ .Values.global.gitlab.urlPrefix }}
-      - name: KEYCLOAK_ADMIN_PASSWORD
-        valueFrom:
-          secretKeyRef:
-            name: {{ template "renku.fullname" . }}-keycloak-password
-            key: keycloak-password
-      - name: KEYCLOAK_ADMIN_USER
-        value: {{ .Values.keycloak.keycloak.username }}
-      - name: KEYCLOAK_URL
-        value: {{ template "http" . }}://{{ .Values.global.renku.domain }}
-    volumeMounts:
-      {{- if .Values.tests.users_json }}
-      - name: demo-users
-        readOnly: true
-        mountPath: /app/users.json
-        subPath: users.json
-      {{- end }}
+  volumes:
+  - name: dshm
+    emptyDir:
+      medium: Memory
+  restartPolicy: Never
   containers:
-  - name: {{ .Release.Name }}-test
+  - name: sbt
     image: "{{ .Values.tests.image.repository }}:{{ .Values.tests.image.tag }}"
     env:
-      - name: RENKU_ENDPOINT
+      - name: RENKU_TEST_URL
         value: {{ template "http" . }}://{{ .Values.global.renku.domain }}
-    lifecycle:
-      preStop:
-        exec:
-          command: ["/tests/demo-cleanup.sh"]
+      - name: RENKU_TEST_FULL_NAME
+        value: "{{ .Values.tests.testUser.fullname }}"
+      - name: RENKU_TEST_USERNAME
+        value: "{{ .Values.tests.testUser.username }}"
+      - name: RENKU_TEST_EMAIL
+        value: "{{ .Values.tests.testUser.email }}"
+      - name: RENKU_TEST_PASSWORD
+        value: "{{ .Values.tests.testUser.password }}"
+      - name: RENKU_TEST_REGISTER
+        value: "1"
```

Comment:
It would be great if all test parameters listed [here](https://github.com/SwissDataScienceCenter/renku/tree/master/acceptance-tests#running-using-sbt) could also be specified through the chart. This will allow us to put all test configuration specific to an instance into the values file and remove some hard-coded defaults (see [this issue](https://github.com/SwissDataScienceCenter/renku/issues/1822)).

Reply:
Done: it is now possible to specify all the env parameters of the acceptance tests through the Helm values.